### PR TITLE
Fixing OCP checks for rh-1.0

### PR DESCRIPTION
**Pull request by anupamtamrakar**
OCP checks were failing because error in audit commands

Also, updated couple of checks text to remove 'Automated' as their type was manual

Please note, there are multiple other checks exist where description text keyword has (Manual) but doesn't have any type

We will plan to update them after discussion

Please review @josedonizetti @chen-keinan @deven0t

### CODE REVIEW

File: cfg/rh-1.0/master.yaml [hunk 1]
```diff
@@ -129,7 +129,7 @@ groups:
         scored: false
 
       - id: 1.1.8
-        text: "Ensure that the etcd pod specification file ownership is set to root:root (Automated)"
+        text: "Ensure that the etcd pod specification file ownership is set to root:root (Manual)"
```
Comment (mozillazg) [inline cfg/rh-1.0/master.yaml:132]: Should update the value of `scored` field to `false` if it should be `Manual`.

File: cfg/rh-1.0/node.yaml [hunk 2]
```diff
@@ -199,7 +199,7 @@ groups:
         scored: true
 
       - id: 4.2.2
-        text: "Ensure that the --authorization-mode argument is not set to AlwaysAllow (Automated)"
+        text: "Ensure that the --authorization-mode argument is not set to AlwaysAllow (Manual)"
```
Comment (mozillazg) [inline cfg/rh-1.0/node.yaml:202]: Ditto.

File: cfg/rh-1.0/node.yaml [hunk 3]
```diff
@@ -346,7 +346,7 @@ groups:
         scored: false
 
       - id: 4.2.9
-        text: "Ensure that the kubeAPIQPS [--event-qps] argument is set to 0 or a level which ensures appropriate event capture (Automated)"
+        text: "Ensure that the kubeAPIQPS [--event-qps] argument is set to 0 or a level which ensures appropriate event capture (Manual)"
```
Comment (mozillazg) [inline cfg/rh-1.0/node.yaml:349]: Ditto.

**Review by mozillazg — CHANGES_REQUESTED** [review]

File: cfg/rh-1.0/master.yaml (hunk 1 above)
Comment (mozillazg) [reply cfg/rh-1.0/master.yaml:132]: I mean this field:
image

**Review by mozillazg — APPROVED** [review]
LGTM, Thanks for your contribution!

**Comment by chen-keinan** [comment]
@anupamtamrakar please rebase with upstream

**Review by chen-keinan — APPROVED** [review]

**Comment by chen-keinan** [comment]
@anupamtamrakar need another rebase

**Pull request merged**